src/actions/layout.js

```diff
@@ -26,12 +31,25 @@ const setHeight = (id, height) => {
   localStorage.setItem(id, String(heightToSave));
 };
 
-const getHeight = id => {
+const getHeight = (id) => {
   const height = localStorage.getItem(id);
   if (height) {
     return parseInt(height, RADIX);
   }
   return null;
 };
 
-export { setHeight, getHeight, openTools, closeTools, setToolsWidth };
+const changeSideBarViewState = (state) => (dispatch) =>
```

Comment:
Maybe change this function's name to match with the type eg. `setSideBarIsOpen`

Reply:
@hasanagh, so this is the only thing missing. Right @pyphilia ?

Reply:
Yes! Will refactor and push the changes!